discovery/seata-discovery-nacos/src/main/java/io/seata/discovery/registry/nacos/NacosRegistryServiceImpl.java

```diff
@@ -235,12 +235,12 @@ private static String getNacosApplicationFileKey() {
     }
 
     private static String getNacosUserName() {
-        return String.join(ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR, ConfigurationKeys.FILE_ROOT_CONFIG, REGISTRY_TYPE,
+        return String.join(ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR, ConfigurationKeys.FILE_ROOT_REGISTRY, REGISTRY_TYPE,
```

Comment:
nacos Config center username is useless?

Reply:
> nacos Config center username is useless?

there is an error here, the registry value should be read

Reply:
ok.